### PR TITLE
Chefs spawning with random produce

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Overview
Chefs now spawn with 8 random, commonly used produce.

## WIPish
There's just 1 bug I need to solve and fix and it should be golden.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -1020,4 +1020,14 @@
 	new /obj/item/weapon/reagent_containers/food/snacks/clam(src)
 	new /obj/item/weapon/reagent_containers/food/snacks/clam(src)
 	new /obj/item/weapon/reagent_containers/food/snacks/clam(src)
-	new /obj/item/weapon/reagent_containers/food/snacks/clam(src)
\ No newline at end of file
+	new /obj/item/weapon/reagent_containers/food/snacks/clam(src)
+
+/obj/item/weapon/storage/box/produce
+	name = "produce box"
+	desc = "A large box of random, leftover produce. Thanks NanoTrasen."
```
Comment (Lohikar) [inline code/game/objects/items/weapons/storage/boxes.dm]: muh current_map.bossname

File: code/game/objects/random/produce.dm [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		"lime" = 0.25,
+		"lemon" = 0.25,
+		"orange" = 0.25,
+		"cocoa" = 0.5,
+		"cherry" = 0.25,
+		"garlic" = 0.5,
+		"onion" = 0.5
+	)
+
+/obj/random_produce/Initialize(var/mapload = 1)
```
Comment (Lohikar) [inline code/game/objects/random/produce.dm]: Why are you specifying a default value for mapload?

Comment (BurgerLUA) [reply code/game/objects/random/produce.dm]: what do you think i know how to code

**Comment by MarinaGryphon** [comment]
I'd object to this on the grounds of Chef powercreep, but like... is this hurting anyone? No. To get more, they still need someone to grow it. It's like lunchboxes but for chefs: a temporary, non-renewable, one time bonus that doesn't mean much in the long run, but can hold you over until you get an actual chef (or in this case, botanist).

**Comment by BurgerLUA** [comment]
!review

**Comment by BurgerLUA** [comment]
This is done btw.

**Pull request merged**